person.rb

```diff
@@ -29,6 +30,10 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- Hey, the add rental method should have a date and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association


```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```